### DISCUSSION TITLE
Smart format applied to Words array

### DISCUSSION THREAD

**Discussion by robfig** [deepgram/community#494] [General help] [answered]
Hi, 
I have been using the Smart Formatted Transcript (Streaming), but I now need to interleave the transcription with those from other audio streams. As a result, I need to use the Words array instead of the formatted / punctuated Transcript. However, the Words array lacks capitalization and punctuation. For example, saying "Testing 123" results in `Transcript: "Testing. 123."` and `Words: ["testing", "123"]` . Is there any way to get the individual words with formatting & punctuation included? 
Thanks,
Rob

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by nikolawhallon** [comment, marked as answer]
There should be both a "word" and a "punctuated_word" field on the `Word` object, it sounds like "punctuated_word" is what you are looking for?

**Reply by robfig** [reply]
:grimacing: 

Totally missed that, thank you!
